console.rst

```diff
@@ -619,6 +619,32 @@ Using Events And Handling Signals
 When a command is running, many events are dispatched, one of them allows to
 react to signals, read more in :doc:`this section </components/console/events>`.
 
+Profiling Commands
+-------------------
```

Comment:
@alamirault the error message from doctor-rst is still wrong (cutted) could you please have a look?

Reply:
I'll add it to my todo list :smiley: